dashboard/modules/snapshot/snapshot_head.py

```diff
@@ -23,6 +25,21 @@
 routes = dashboard_optional_utils.ClassMethodRouteTable
 
 
+@dataclasses.dataclass
+class RayActivityResponse:
+    """
+    Dataclass used to inform if a particular Ray component can be considered
+    active, and metadata about observation.
+    """
+
+    # Whether the corresponding Ray component is considered active
+    is_active: bool
+    # Why the Ray component is considered active
+    reason: str
```

Comment:
Is it None if is_active == False? Otherwise the comment & code seems to be wrong from the description. 

Maybe "The reason if `is_active` flag"?